service/conf.ml

```diff
@@ -69,6 +69,8 @@ let platforms =
       v "centos"   Builders.amd3 "centos-8"      default_compiler;
       v "fedora"   Builders.amd3 "fedora-31"     default_compiler;
       (* oraclelinux doesn't work in opam 2 yet *)
+      (* Variants *)
+      v "4.10+32bit" Builders.amd4 "debian-10" "4.10+32bit";
```

Comment:
```suggestion
      v (default_compiler ^ "+32bit") Builders.amd4 "debian-10" (default_compiler ^ "+32bit");
```

At some point, `default_compiler` will probably be determined dynamically to be the highest OCaml version supported by the project. I think we might as well avoid hard-coding `4.10` here.